### PR TITLE
Feature/505 read only settings page

### PR DESCRIPTION
**Pull request by skuhlmann**
## GitHub Issue

[GitHub issue.](https://github.com/HausDAO/daohaus-monorepo/issues/505)

## Changes

- new page with all the settings.
- moved tooltip content to the common-utils

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: apps/core-app/src/components/GovernanceSettings.tsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  const { daochain } = useParams();
+  const networkData = useMemo(() => {
+    if (!daochain) return null;
+    return getNetwork(daochain);
+  }, [daochain]);
+
+  return (
+    <GovernanceContainer>
+      <GovernanceCardHeader>
+        <H3>Governance</H3>
```
Comment (brossetti1) [inline apps/core-app/src/components/GovernanceSettings.tsx:73]: this suppose to be `Governance Settings`?
Screen Shot 2022-08-05 at 2 35 53 PM

File: apps/core-app/src/pages/Settings.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            <GovernanceSettings dao={dao} />
+          </SettingsContainer>
+
+          <SettingsContainer>
+            <ShamanSettings dao={dao} />
+          </SettingsContainer>
+        </>
+      )}
+    </SingleColumnLayout>
+  );
```
Comment (brossetti1) [inline apps/core-app/src/pages/Settings.tsx:41]: nice

File: apps/core-app/src/components/GovernanceSettings.tsx (hunk 1 above)
Comment (skuhlmann) [reply apps/core-app/src/components/GovernanceSettings.tsx:73]: Yep - updating

**Review by brossetti1 — APPROVED** [review]
tried to load up the UI to check out layout and whatnot, not sure if im on any doas but was not able to visit the settings page. maybe we should share our addresses so when anyone is making a dao, we all get put on them so we have some data to work with?

anyways, approving

**Comment by skuhlmann** [comment]
> tried to load up the UI to check out layout and whatnot, not sure if im on any doas but was not able to visit the settings page. maybe we should share our addresses so when anyone is making a dao, we all get put on them so we have some data to work with?
> 
> anyways, approving

good idea - been testing this this dao:

http://localhost:3000/#/molochv3/0x5/0x24357654d64da97a55fb2f73c5b66d1927ab1e4c

**Pull request merged**